protobuf/filedescriptorsets_test.go

```diff
@@ -0,0 +1,99 @@
+package protobuf
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewDescriptorProviderFileDescriptorSetBins(t *testing.T) {
+	tests := []struct {
+		name         string
+		fileNames    []string
+		errMsg       string
+		lookupSymbol string
+		symbolErrMsg string
+	}{
+		{
+			name:         "pass",
+			fileNames:    []string{"../testdata/protobuf/simple/simple.proto.bin"},
```

Comment:
every path here is prefixed with `../testdata/protobuf/` -- maybe the test execution logic should add that prefix and keep the test-table cleaner?